utilities/hooks/useForm.ts

```diff
@@ -0,0 +1,44 @@
+import { ChangeEventHandler, useReducer } from 'react'
+import { FormProps } from '../types/formTypes'
+
+const RESET = 'reset'
+const UPDATE = 'update'
+interface ActionState {
+    type: string
+    payload?: any
```

Comment:
Actually...need to add something here 😄 Very good that you used the conditional `?`. BUT, for when it isn't conditional, we should be assigning proper types. Since this is always a form state any time it's used, it's safe to say 99.99% of the time we can expect the payload to be 1 of 2 things:

- Name and value pair for `UPDATE`. Of which, the name is always a type `string` and value is a type of multiple possible types, as we expand the form it'll grow. for now, the only type accessible is `string`. 
- Bulk import (which isn't needed yet but will be used at some point) that will just reassign the FormProps type

I'd like to try and keep our types as tight as possible, as it's best practice and great habit. As few `any` types as possible is great